### ISSUE TITLE
Update documentation for Python 3

### ISSUE THREAD

**Issue by snorfalorpagus** [Toblerity/Shapely#749]
There are some examples in the documentation that are valid in Python 2, but not in Python 3. For example:

https://github.com/Toblerity/Shapely/blob/master/docs/manual.rst#general-attributes-and-methods

```python
>>> print Point(0, 0).geom_type
Point
```

These examples should be updated to use Python 3 syntax. In the case of print statements, I'd be OK with just returning a string instead, e.g.

```python
>>> Point(0, 0).geom_type
'Point'
```

**Comment by clncy** [comment]
@snorfalorpagus I can't see any PRs addressing this, shall I create one?

**Comment by snorfalorpagus** [comment]
@clncy PR would be very welcome! I'd be happy to review it.

**Comment by clncy** [comment]
@snorfalorpagus Thank you very much. I'd like to contribute to Shapely, so if there are any more documentation issues, missing test cases or code changes I can address, feel free to mention me.

**Comment by snorfalorpagus** [comment]
Closed by #776.
